### DISCUSSION TITLE
`GLIBCXX_3.4.22' not found

### DISCUSSION THREAD

**Discussion by Enzo-bit-beep** [PaddlePaddle/PaddleOCR#13016] [Q&A] [answered]
在import paddleocr时遇到
ImportError: /usr/local/gcc-5.2.0/lib64/libstdc++.so.6: version `GLIBCXX_3.4.22' not found (required by /usr/local/anaconda3/envs/py39/lib/python3.9/site-packages/paddle/base/libpaddle.so)
但我确实有GLIBCXX_3.4.22
谁知道怎么解决啊！！

**Comment by tink2123** [comment, marked as answer]
you'll need to add the path to the lib folder in anaconda to LD_LIBRARY_PATH：

export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:{/your/path/to/conda/env/lib}
